apps/web/src/pages/templates/WorkflowListPage.tsx

```diff
@@ -71,7 +71,7 @@ const columns: IExtendedColumn<INotificationTemplateExtended>[] = [
     maxWidth: 125,
     Cell: withCellLoading(({ row: { original } }) => (
       <>
-        {original.draft ? (
+        {!original.active ? (
```

Comment:
after change users may still have those "broken" templates with active=true and draft=true.

so without this change on frontned they could still see dual state :)